src/content/docs/en/recipes/docker.mdx

```diff
@@ -148,10 +148,10 @@ WORKDIR /app
 COPY package.json package-lock.json ./
```

Comment:
Suggestion/Nitpick: While `./` works fine, perhaps `/app/` would convey more information without relying on the previous state of the `WORKDIR`.